Library/Homebrew/tab.rb

```diff
@@ -292,6 +298,7 @@ def to_json
       "time" => time,
       "source_modified_time" => source_modified_time.to_i,
       "HEAD" => self.HEAD,
+      "alias_path" => alias_path && alias_path.to_s,
```

Comment:
Probably worth doing `(alias_path.to_s if alias_path)` just to match style below.